packages/x-date-pickers/src/PickersDay/PickersDay.tsx

```diff
@@ -67,6 +67,10 @@ export interface PickersDayProps<TDate>
    * @default false
    */
   today?: boolean;
+  /**
+   * The days currently selected.
```

Comment:
When would it simplify to have this default ?
We have just one component calling `PickersDay` (`DayPicker`) and he always has an array to pass.

Reply:
I've just looked through changes and there were a few files with changes only adding `selectedDays={[]}`, but yeah, feel free to ignore this comment as it's more of a testing cases rather than user facing API. 👌